Backend/app/users/management/commands/create_sample_data.py

```
@@ -0,0 +1,154 @@
+
+    def handle(self, *args, **options):
+        self.stdout.write('Creating sample data...')
+        
+        # Create therapists
+        therapists = []
+        for i in range(options['therapists']):
+            therapist_user = User.objects.create_user(
+                username=f'therapist{i+1}@example.com',
+                email=f'therapist{i+1}@example.com',
+                password='password123',
+                first_name=f'Dr. Therapist',
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)